c++/src/kj/string-test.c++

```diff
@@ -497,6 +497,10 @@ struct OnlyMoves {
   bool operator==(const OnlyMoves&) const = default;
 };
 
+struct Stringable {
```

Comment:
We should also restore `TEST(String, ToString)` that was removed in the previous commit.

Reply:
Definitely